### PR TITLE
add cache headers

### LINKED ISSUE #13: please set cache headers
Thank you for this package, it makes setting up a vanity import path pretty easy indeed.

I noticed that no cache headers are set in any responses:

```
% curl -v <url>
< HTTP/2 200 
< content-type: text/html; charset=utf-8
< x-cloud-trace-context: e11dd858ab85c26bd488a27ac5deb139;o=1
< date: Sun, 24 Sep 2017 07:38:12 GMT
< server: Google Frontend
< content-length: 566
< alt-svc: quic=":443"; ma=2592000; v="39,38,37,35"
< 
<!DOCTYPE html>
[…]
```

When using CloudFlare in front of AppEngine, it is beneficial to set caching headers to reduce traffic to AppEngine. Currently, every request needs to go to AppEngine, even though the response will be exactly the same.

I suggest that cache headers should be included with every response, with the default cache time being 24 hours or so (configurable). This should be a good default, given how rarely the config changes in practice.

See https://support.cloudflare.com/hc/en-us/articles/202775670-How-Do-I-Tell-Cloudflare-What-to-Cache- for which headers CloudFlare respects.

Thank you for considering.

### PR DESCRIPTION
**Pull request by zombiezen**
Fixes #13

### CODE REVIEW

File: handler.go [hunk 1]
```diff
@@ -39,8 +41,9 @@ type pathConfig struct {
 
 func newHandler(config []byte) (*handler, error) {
 	var parsed struct {
-		Host  string `yaml:"host,omitempty"`
-		Paths map[string]struct {
+		Host     string `yaml:"host,omitempty"`
+		CacheAge *int64 `yaml:"cache_age,omitempty"`
```
Comment (rakyll) [inline handler.go]: Should we call it cache_max_age or just cache_secs or something in the lines of that? People already are familiar with max age and can guess the unit.

File: handler_test.go [hunk 2]
```diff
@@ -227,3 +231,46 @@ func TestPathConfigSetFind(t *testing.T) {
 		}
 	}
 }
+
+func TestCacheHeader(t *testing.T) {
+	tests := []struct {
+		name         string
+		config       string
+		cacheControl string
+		configErr    bool
```
Comment (rakyll) [inline handler_test.go]: you don't use configErr.

File: handler.go (hunk 1 above)
Comment (zombiezen) [reply handler.go]: Done.

File: handler_test.go (hunk 2 above)
Comment (zombiezen) [reply handler_test.go]: Quite right. Removed.

**Pull request merged**